model/project.rb

```diff
@@ -10,6 +10,7 @@ class Project < Sequel::Model
 
   many_to_many :vms, join_table: AccessTag.table_name, left_key: :project_id, right_key: :hyper_tag_id
   many_to_many :private_subnets, join_table: AccessTag.table_name, left_key: :project_id, right_key: :hyper_tag_id
+  many_to_many :postgres_databases, class: PostgresResource, join_table: AccessTag.table_name, left_key: :project_id, right_key: :hyper_tag_id
```

Comment:
Should we use `postgres_resources` to match with the model name?

Reply:
I changed this to postgres_resources, but at some point, there needs to be transition to postgres_databases because it is what we show in the UI. I moved that transition to routes/web/project/postgres.rb